lib/fitgem_oauth2/client.rb

```diff
@@ -51,6 +51,16 @@ def refresh_access_token(refresh_token)
       JSON.parse(response.body)
     end
 
+    def revoke_token(token)
+      response = connection.post('/oauth2/revoke') do |request|
+        encoded = Base64.strict_encode64("#{@client_id}:#{@client_secret}")
+        request.headers['Authorization'] = "Basic #{encoded}"
+        request.headers['Content-Type'] = 'application/x-www-form-urlencoded'
+        request.params['token'] = token
+      end
+      JSON.parse(response.body)
+    end
+
```

Comment:
Thanks, @dav-armour for your PR. Can you please also add a unit test for the method?